src/main/java/htsjdk/samtools/cram/structure/CramCompressionRecord.java

```diff
@@ -210,6 +210,17 @@ public void setSegmentUnmapped(final boolean segmentUnmapped) {
         flags = segmentUnmapped ? flags | SEGMENT_UNMAPPED_FLAG : flags & ~SEGMENT_UNMAPPED_FLAG;
     }
 
+    /**
+     * Does this record have a valid placement / alignment location?
+     * It must have both a valid reference sequence ID and alignment start position to qualify.
```

Comment:
Might be worth explicitly stating that this returns the placed status independent of mapped status (that is, it can return true even when the record is mapped).